static/app/components/profiling/flamegraph/collapsibleTimeline.tsx

```diff
@@ -71,7 +71,7 @@ export function CollapsibleTimelineLoadingIndicator({size}: {size?: number}) {
 const CollapsibleTimelineContainer = styled('div')`
   position: relative;
   width: 100%;
-  height: 100%;
+  height: calc(100% - 20px);
```

Comment:
Without this, the chart is slightly higher than the timeline as the label height is not accounted for